### DISCUSSION TITLE
Unité liste de reflet

### DISCUSSION THREAD

**Discussion by StephanBerio** [witsa/synapps#713] [Q&A]
Bonjour,
Dans une variable, notamment d’import en communication Modbus, est spécifié en Paramètres l’Unité.
Cette variable (avec d’autres) est mise dans un Ensemble Et cet ensemble remonte en Synapps par une Liste de Reflets.
Quand on modifie l’unité dans les Paramètres de la variable du Redy (exemple passage d’une énergie de Wh en kWh ou MWh), 
L’unité n’est pas modifiée en visualisation Synapps de la Liste de Reflets…
 
Merci pour votre aide
Bien Cordialement
BÉRIO Stéphan

**Comment by WITNicolas** [comment]
Doublon de : https://github.com/witsa/synapps/issues/562
JYK
OV

Stephan, les unités avec l'acteur Liste d'Etat remontent bien.
